src/main/webapp/content/scss/quiz.scss

```diff
@@ -324,7 +324,7 @@
         position: absolute;
         top: 20px;
         right: 20px;
-        max-width: 15%;
+        max-width: 20%;
```

Comment:
please test how this looks on smaller screens (simply reduce the width of the browser window).